screwdriver.yaml

```
@@ -9,7 +9,7 @@ shared:
     steps:
       - .: &install-node
           install-node: |
-            sd-cmd exec screwdriver/install-nodejs@latest $NODE_VERSION
+            sd-cmd exec screwdriver/install-nodejs@1.0.4 $NODE_VERSION
```

1.0.4 is latest according to https://cd.screwdriver.cd/commands/screwdriver/install-nodejs
```suggestion
            sd-cmd exec screwdriver/install-nodejs@latest $NODE_VERSION
```

---

Updated